src/overview/actions.js

```diff
@@ -20,6 +20,13 @@ export const setEndDate = createAction('overview/setEndDate')
 export const hideResultItem = createAction('overview/hideResultItem')
 export const showDeleteConfirm = createAction('overview/showDeleteConfirm')
 export const hideDeleteConfirm = createAction('overview/hideDeleteConfirm')
+export const showFilter = createAction('overview/showFilter')
+export const setShowOnlyBookmarks = createAction(
+    'overview/setShowOnlyBookmarks',
+)
+export const showOnlyBookmarksEnhancer = createAction(
+    'overview/showOnlyBookmarksEnhancer',
+)
```

Comment:
`showFilter` and `setShowOnlyBookmarks` seem to be toggle-type actions (they just invert the current state). Better to call these `toggleShowFilter` and `toggleBookmarkFilter`. "enhancer" is a particular type of extension you can make to a redux store, so instead of calling the last action `showOnlyBookmarksEnhancer`, that would be better as `setShowOnlyBookmarks` (as it is explicitly "setting" a state value, while the others are just "toggling")

Reply:
by `showOnlyBookmarksEnhancer` we are setting the value of checkbox bu using `params`.

Reply:
Ah sorry I didn't mean you need to remove both actions and merge into one. You could keep both, but one acts as a "toggle" (no input params/payload, just inverts current state), the other acts as a "setter" (input value becomes new state). But don't worry too much if this is working fine :)